docs/how_we_work.md

```diff
@@ -107,3 +107,33 @@ Open source is awesome, and sometimes we get pull requests from non-Mozillians.
 
 TODO: should we ever consider a [Developer Certificate of Origin](https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md#signing-your-commits). 
 
+### Github Project Columns Defined
+
+##### Backlog
+
+A list of possible tasks. Tasks in this column are not actively being worked on. In our weekly triage meeting this list is briefly discussed and if needed, prioritized. 
+
+TODO: We should have an icebox.
```

Comment:
How about we use https://github.com/mozmar/infra/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20no%3Aproject for an icebox?